src/main/kotlin/no/nav/familie/ef/sak/api/journalføring/JournalføringController.kt

```diff
@@ -13,34 +15,40 @@ import org.springframework.web.bind.annotation.*
 @RequestMapping("/api/journalpost")
 @ProtectedWithClaims(issuer = "azuread")
 @Validated
-class JournalføringController(val journalføringService: JournalføringService, val pdlClient: PdlClient) {
+class JournalføringController(val journalføringService: JournalføringService, val pdlClient: PdlClient, private val tilgangService: TilgangService) {
 
     @GetMapping("/{journalpostId}")
     fun hentJournalPost(@PathVariable journalpostId: String): Ressurs<JournalføringResponse> {
-        // TODO: Tilgangskontroll til person
-        val journalpost = journalføringService.hentJournalpost(journalpostId)
-        val personIdent = journalpost.bruker?.let {
-            when (it.type) {
-                BrukerIdType.FNR -> it.id
-                BrukerIdType.AKTOERID -> pdlClient.hentPersonident(it.id).hentIdenter.identer.first().ident
-                BrukerIdType.ORGNR -> error("Kan ikke hente journalpost= ${journalpostId} for orgnr")
-            }
-        } ?: error("Kan ikke hente journalpost= ${journalpostId} uten bruker")
+        val (journalpost, personIdent) = finnJournalpostOgPersonIdent(journalpostId)
+        tilgangService.validerTilgangTilPersonMedBarn(personIdent)
         return Ressurs.success(JournalføringResponse(journalpost, personIdent))
     }
 
     @GetMapping("/{journalpostId}/dokument/{dokumentInfoId}")
     fun hentDokument(@PathVariable journalpostId: String, @PathVariable dokumentInfoId: String): Ressurs<ByteArray> {
-        // TODO: Tilgangskontroll til person
+        val (_, personIdent) = finnJournalpostOgPersonIdent(journalpostId)
```

Comment:
Denne er fin :) , men hvis vi får mange flere (enn to) så kan vi kanskje vurdere å lage en ny metode finnPersonIdent(journalpostId)? Eller ikke - bare tenker høyt.

Reply:
Godt poeng